example/app.js

```diff
@@ -9,6 +9,6 @@ const app = multiUserDevServer(username => {
     // What to respond with for `GET /:username` (optional)
     successResponse: `Bundle completed in ${__dirname}/${username}`,
   };
-});
+}, /* expireAfter */ 10);
```

Comment:
10 seconds? That's it? It takes longer than that to reload a page sometimes...

Reply:
This is the toy example app, it's meant to show off the functionality (see the webpack configs in this dir), this isn't the app or config that we use.